docs/apache-airflow-providers-edge/edge_executor.rst

```diff
@@ -209,6 +210,50 @@ could take thousands of tasks without a problem), or from an environment
 perspective (you want a worker running from a specific location where required
 infrastructure is available).
 
+Capacity handling
```

Comment:
Can you add a note that if not provided default capacity is counted as 1?

Also worthwile to mention that integers are supported, not like K8s where you could reserver `cpu=300millis`